cache-config/testing/ort-tests/tcdata/todb.go

```diff
@@ -169,7 +171,7 @@ func (r *TCData) SetupTmusers(db *sql.DB) error {
 	sqlStmt := `
 INSERT INTO tm_user (username, local_passwd, role, tenant_id) VALUES ('` + r.Config.TrafficOps.Users.Disallowed + `','` + encryptedPassword + `', 1, 1);
 INSERT INTO tm_user (username, local_passwd, role, tenant_id) VALUES ('` + r.Config.TrafficOps.Users.ReadOnly + `','` + encryptedPassword + `', 2, 1);
-INSERT INTO tm_user (username, local_passwd, role, tenant_id) VALUES ('` + r.Config.TrafficOps.Users.Operations + `','` + encryptedPassword + `', 3, 1);
+INSERT INTO tm_user (username, local_passwd, role, tenant_id) VALUES ('` + r.Config.TrafficOps.Users.Operations + `','` + encryptedPassword + `', 4, 1);
```

Comment:
You changed the "operations" Role above, but this changes the "operations" user to have the "admin" Role instead of the "operations" Role at all, so those changes are totally unused.